PWGGA/GammaConv/AliAnalysisTaskConversionTree.cxx

```diff
@@ -52,11 +52,14 @@ AliAnalysisTaskConversionTree::AliAnalysisTaskConversionTree() : AliAnalysisTask
   ffillTree(-100),
   fOutputList(NULL),
 
+  hInvMass(NULL),
```

Comment:
Dear Ewa, please undo all your changes to this AnalysisTask and its header. Your changes were for local testing only.